src/sections/Community/Handbook/repo-data.js

```diff
@@ -339,16 +391,22 @@ export const repo_data = [
         project: "SMP Action",
         image: github,
         language: "shell",
-        description: "GitHub Action to run SMP Performance Benchmarks on CI/CD pipelines.",
-        repository: "https://github.com/layer5io/meshery-smp-action"
+        maintainers_name: ["Vacant"],
+        link: ["#"],
+        description:
+          "GitHub Action to run SMP Performance Benchmarks on CI/CD pipelines.",
+        repository: "https://github.com/layer5io/meshery-smp-action",
       },
       {
         project: "Meshery-Perf",
         image: meshery,
         language: "Golang",
-        description: "meshery-perf is a Meshery component used to run performance tests",
-        repository: "https://github.com/meshery/meshery-perf"
-      }
+        maintainers_name: ["Vacant"],
```

Comment:
@gyohuangxin?  - - he'll have to confirm...

Reply:
I'll change it after @gyohuangxin confirms.